app/pages/project/stats/stats.cjsx

```diff
@@ -164,9 +164,9 @@ ProjectStatsPage = React.createClass
   render: ->
     progress = @workflowInfo()
     #Dates for gap in classification stats
-    classificationGap = ['2016-01-13T00:00:00.000Z', '2016-02-07T00:00:00.000Z']
+    classificationGap = ['2015-06-30T00:00:00.000Z', '2016-06-09T00:00:00.000Z']
```

Comment:
The first date is the start of the gap.  Since all data before Jun 2016 is gone the first date is now the launch of the project builder (in 2015).